kolibri/plugins/coach/assets/src/views/common/resourceSelection/UpdatedResourceSelection.vue

```diff
@@ -81,6 +83,11 @@
         required: false,
         default: () => [],
       },
+      selectAllRules: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
```

Comment:
This appears to expect an array of functions, might be good to add a validator to that effect. Could use lodash's [isFunction](https://lodash.com/docs/#isFunction) on each item given to make it simple

Reply:
Also it might be good to note in a comment here that it  specifically expects a set of predicate functions that should return bools

Reply:
Thanks! Just did it! :)